.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Links to any relevant github issues:
+
+- [Issue description](http://example.com/org/repo/issues/411). Any other useful information.
+
+### What's in the the PR
+
+- [ ] A description of the changes proposed in the pull request.
+  - [ ] Any areas or issues reviewer should pay attention to? (comments in diff okay).
+- [ ] Update `/docs` to include, whenever possible, a new, suitable recommendation about how to use
+    the feature.
+- [ ] Extra Credit: [Confirm it runs and tests pass on the Rubies specified in the Travis
```

Comment:
oh, needs to be `../.travis.yml` now.  Thanks, good question.  Definitely needs simplification or removal.

This is my stupid multi rails test https://github.com/rails-api/active_model_serializers/blob/master/CONTRIBUTING.md#running-tests

```bash
for version in 4.0 4.1 4.2 master; do
  export RAILS_VERSION="$version"
  rm -f Gemfile.lock
  bundle check || bundle --local || bundle
  bundle exec rake test
  if [ "$?" -eq 0 ]; then
    # green in ANSI
    echo -e "\033[32m **** Tests passed against Rails ${RAILS_VERSION} **** \033[0m"
  else
    # red in ANSI
    echo -e "\033[31m **** Tests failed against Rails ${RAILS_VERSION} **** \033[0m"
  fi
  unset RAILS_VERSION
done
```

I also like https://github.com/grosser/wwtd